src/Frame.js

```diff
@@ -0,0 +1,36 @@
+var Frame = function() {
+  this.score = 0;
+  this.Pin = 10;
+};
+
+  Frame.prototype.score = function() {
+    this.score = this.roll1 + this.roll2;
+  };
+
+  Frame.prototype.isSpare = function() {
+  if (roll1 + roll2) === 10 {
+    return true
+  } else {
+    return false
+  }
+
+  Frame.prototype.isStrike = function() {
+    if roll1 === 10 {
+      return true
+    } else {
+      return false
+    }
+  };
```

Comment:
Actually, if you find yourself explicitly returning true and false, your code probably needs to just be:
```javascript
Frame.prototype.isStrike = function() {
  return roll1 === 10;
}
```